hack/builder/publish.sh

```diff
@@ -1,11 +1,26 @@
 #!/usr/bin/env bash
+set -ex
 
 SCRIPT_DIR="$(
     cd "$(dirname "$BASH_SOURCE[0]")"
     pwd
 )"
 
+trap 'cleanup' EXIT
+
+cleanup() {
+    rm manifests/ -rf || true
+}
+
 . ${SCRIPT_DIR}/version.sh
 
-docker tag kubevirt/builder:${VERSION} docker.io/kubevirt/builder:${VERSION}
-docker push docker.io/kubevirt/builder:${VERSION}
+cleanup
+
+for ARCH in ${ARCHITECTURES}; do
+    docker push kubevirt/builder:${VERSION}-${ARCH}
+    TMP_IMAGES="${TMP_IMAGES} kubevirt/builder:${VERSION}-${ARCH}"
+done
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
@rmohr Which experimental feature are we using?

Reply:
Just pushed something for testing prow. These changes are actually part of #3045. Building multi-arch manifests is an experimental feature on the docker client.